detox/src/ios/earlgreyapi/GREYActions.js

```diff
@@ -79,54 +79,6 @@ simulate a long press.
     };
   }
 
-  /*Returns an action that holds down finger for specified @c duration to simulate a long press.
-
-@param duration The duration of the long press.
-
-@return A GREYAction that performs a long press on an element.
-*/static actionForLongPressWithDuration(duration) {
```

Comment:
@rotemmiz I double checked it, it is not included in the [switch statement in method invocation](https://github.com/wix/detox/blob/master/detox/ios/Detox/MethodInvocation.m#L34) Therefore we need to remove it for now. I will add this functionality while extending `MethodInvocation` in the same manner. That way the commits are sane and make sense 👍